### PR TITLE
hdrop: 0.2.4 -> 0.3.0

### PR DESCRIPTION
**Pull request by Schweber**
Changed `hyprwm/contrib` README description of hdrop

added --insensitive and --verbose
Improved documentation.

### CODE REVIEW

File: hdrop/hdrop [hunk 1]
```diff
@@ -26,45 +35,76 @@ print_help() {
...
--*)
-  notify "hdrop: unrecognized flag" "Run 'hdrop -h' for more information"
-  print_help
-  exit 1
-  ;;
-esac
+# shopt -s extglob
+
+while true; do
+  case "$1" in
```
Comment (fufexan) [inline hdrop/hdrop:64]: I think this is getting long enough that `getopt` would be better suited for parsing arguments.

Comment (Schweber) [reply hdrop/hdrop:64]: I thought about that but then wasn't sure how to separate hdrop flags from the flags of the program to be run by hdrop. How would you do that?

Comment (fufexan) [reply hdrop/hdrop:64]: There's a general consistency between programs I use that if you pass `--`, flags after that are no longer the calling program's business. Maybe something like that can be handled.

Comment (Schweber) [reply hdrop/hdrop:88]: I think i found a solution...

**Review by Schweber — COMMENTED** [review]
There may be a more elegant way but i think that it works like this

**Review by fufexan — APPROVED** [review]
Alright, if it works, LGTM.

**Pull request merged**